### PR TITLE
Update rubygem-foreman_puppet to 1.0.3

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Latest bug fixes

### CODE REVIEW

File: packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec [hunk 1]
```diff
@@ -8,7 +8,7 @@
 %global foreman_min_version 2.4
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.1
+Version: 1.0.2
 Release: 1%{?foremandist}%{?dist}
 Summary: Adds puppet ENC features
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:13]: I think this isn't entirely accurate anymore since it does more.

**Review by ekohl — COMMENTED** [review]
Looks good, but I think you're already working on a next release. Should we wait for that?

File: packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec (hunk 1 above)
Comment (ezr-ondrej) [reply packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:13]: Happy to change that in the gem, but now it is the gem's summary :shrug:

**Comment by ezr-ondrej** [comment]
I've updated this to the 1.0.3, since it contains few more bugfixes

**Pull request merged**